contracts/core/extensions/CoreAccounting.sol

```diff
@@ -215,8 +199,21 @@ contract CoreAccounting is
         uint[] _quantities
     )
         internal
-        isValidBatchTransaction(_tokens, _quantities)
     {
+        // Confirm and empty _tokens array is not passed
+        require(
```

Comment:
for style, since we are no longer passing in messages, let's have the check be all in one line